nucypher/cli/commands/ursula.py

```diff
@@ -442,3 +449,171 @@ def _pre_launch_warnings(emitter, dev, force):
+                 force: bool,
+                 ) -> None:
+        self.worker_path = Path(worker_path)
+        self.keystore_path = Path(keystore_path)
+        self.backup_path = Path(backup_path)
+        self.password = password.encode()
+        self.force = force
+
+
+
+def _default_keystore_path():
+    # Reference: https://ethdocs.org/en/latest/account-management.html#manual-backup-restore
+    keystore_paths = {
+        'Windows': Path('%appdata%\Roaming\Ethereum\keystore'),
+        'Linux': Path('.ethereum/keystore'),
+        'Darwin': Path('Library/Ethereum/keystore'),
+    }
+    system = platform.system()
+    if system in keystore_paths.keys():
+        return Path.home() / keystore_paths[system]
+    return Path.home()
+
+
+def _deactivate_prompts(ctx, _param, value):
+    if value:
+        for p in ctx.command.params:
+            if isinstance(p, click.Option):
+                p.prompt = None
+    return value
+
+backup_cli_options = group_options(
+    BackupCliOptions,
+    password=click.option('--password',
+                          help="Enter backup password",
+                          prompt="Enter backup password",
+                          hide_input=True,
+                          required=True,
+                          confirmation_prompt=True),
+    worker_path=click.option('--worker-path',
+                             help=f"Path to Ursula worker directory [{DEFAULT_CONFIG_ROOT}]",
+                             prompt="Path to Ursula worker directory",
+                             type=click.Path(),
+                             required=True,
+                             default=DEFAULT_CONFIG_ROOT),
+    keystore_path=click.option('--keystore-path',
+                               help=f"Path to Ursula keystore directory [{_default_keystore_path()}]",
+                               prompt="Path to Ursula keystore directory",
+                               type=click.Path(),
+                               required=True,
+                               default=_default_keystore_path()),
+    backup_path=click.option('--backup-path',
+                             help="Path to backup file",
+                             prompt="Path to backup file",
+                             type=click.Path(dir_okay=False),
+                             default=Path("./ursula-backup.zip"),
+                             required=True),
+    force=click.option('--force', 
+                       help="Disable prompts", 
+                       default=False,
+                       is_eager=True, 
+                       is_flag=True, 
+                       callback=_deactivate_prompts)
+)
+
+def _prompt_overwrite(path: Path, force: bool):
+    if path.exists():
+        prompt = f"Path already exists: {path.absolute()}"
+        if force:
+            click.echo(f'{prompt} - Overwriting.')
+        else:
+            click.echo(f'{prompt}\n Overwrite?')
+            click.confirm(prompt, abort=True)
```

Comment:
Getting prompt text twice here:
(backup)
```bash
Path already exists: /Users/derek/Documents/Github/repos/forks/piotr/nucypher/ursula-backup.zip
 Overwrite?
Path already exists: /Users/derek/Documents/Github/repos/forks/piotr/nucypher/ursula-backup.zip [y/N]:
```

(restore)
```
Path already exists: /Users/derek/Documents/Github/repos/forks/piotr/nucypher/keystore
 Overwrite?
Path already exists: /Users/derek/Documents/Github/repos/forks/piotr/nucypher/keystore [y/N]: y
```

```suggestion
            click.confirm(f'{prompt} - Overwrite?', abort=True)
```